### ISSUE TITLE
Navbar Linebreak on Mobile

### ISSUE THREAD

**Issue by mtwente** [digihistch24/book-of-abstracts#45]
### Current Behavior

On smaller screens, the search box logo (the last one) is placed on a new line, when the navbar should have one line only:
<img width="493" alt="Bildschirmfoto 2024-08-21 um 21 28 41" src="https://github.com/user-attachments/assets/2eb5ac36-3a80-4efa-98e5-f8e49b103f10">

On medium screens, both the github icon and the darkmode toggle are not aligned properly. They should be placed on the far right, next to the search button.

<img width="837" alt="Bildschirmfoto 2024-08-21 um 21 29 45" src="https://github.com/user-attachments/assets/0bbc79b2-5736-4a49-8669-7c90e0d9bd1a">

**Comment by maehr** [comment]
A quick fix is to turn of dark mode. To fix it we would need to fix the quarto template. It looks a little messy, especially the `aa-DetachedSearchButton` class...

**Comment by maehr** [comment]
Alternatively we could uncomment the following section in `_quarto.yml`

```yml
    # tools:
    #   - icon: github
    #     url: https://github.com/digihistch24/book-of-abstracts/
    #     target: _blank
```

**Comment by mtwente** [comment]
Yes. Each abstract page has a separate link to edit on github anyway.
But also, neither dark mode nor this icon are really necessary...

**Comment by maehr** [comment]
Fair point. Let's discuss this later today.
